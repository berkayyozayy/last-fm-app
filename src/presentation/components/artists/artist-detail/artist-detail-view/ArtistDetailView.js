import TopAlbums from "../albums/TopAlbums";
import TopTracks from "../tracks/TopTracks";
import styles from "./ArtistDetailView.module.css";

function ArtistDetailView() {
  return (
    <div className={styles.container}>
      <div className={styles["album-list"]}>
        <TopAlbums />
      </div>
      <div className={styles["track-list"]}>
        <TopTracks />
      </div>
    </div>
  );
}

export default ArtistDetailView;
