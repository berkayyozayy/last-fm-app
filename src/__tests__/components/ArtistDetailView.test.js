import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import ArtistDetailView from "../../presentation/components/artists/artist-detail/artist-detail-view/ArtistDetailView";

describe("ArtistDetailView component", () => {
  test("it should render", () => {
    const { container } = render(
      <MemoryRouter>
        <ArtistDetailView />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
