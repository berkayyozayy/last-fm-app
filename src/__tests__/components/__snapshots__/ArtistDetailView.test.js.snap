// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistDetailView component it should render 1`] = `
<div>
  <h1
    class="title"
  >
     's Best Tracks & Albums
  </h1>
  <div
    class="container"
  >
    <div
      class="album-list"
    >
      <div
        class="loading"
      >
        <div
          class="circle"
        />
      </div>
    </div>
    <div
      class="track-list"
    >
      <div
        class="loading"
      >
        <div
          class="circle"
        />
      </div>
    </div>
  </div>
</div>
`;
